pkg/kn/commands/revision/delete_test.go

```diff
@@ -60,3 +60,22 @@ func TestRevisionDelete(t *testing.T) {
 	}
 	assert.Check(t, util.ContainsAll(output, "Revision", revName, "deleted", "namespace", commands.FakeNamespace))
 }
+
+func TestMultipleRevisionDelete(t *testing.T) {
+	revName1 := "foo-12345"
+	revName2 := "foo-67890"
+	revName3 := "foo-abcde"
+	action, _, output, err := fakeRevisionDelete([]string{"revision", "delete", revName1, revName2, revName3})
```

Comment:
Take a look at this unit tests framework we're using now https://github.com/knative/client/blob/master/pkg/kn/commands/service/service_update_mock_test.go#L32

We should eventually streamline all the unit tests to use single framework.